.travis.yml

```diff
@@ -1,11 +1,20 @@
-language: sh
+language: bash
 
-before_script:
-  - wget "http://ftp.debian.org/debian/pool/main/s/shellcheck/shellcheck_0.3.7-5_amd64.deb"
-  - sudo dpkg -i "shellcheck_0.3.7-5_amd64.deb"
+# Use container-based infrastructure for quicker build start-up
+sudo: false
+
+addons:
+  apt:
+    sources:
+    - debian-sid # Get shellcheck from the Debian repo
```

Comment:
I looked into this more: the source alias is defined in https://github.com/travis-ci/apt-source-whitelist/blob/master/ubuntu.json#L87-L91 . `debian-sid` is the only Debian offered.

Carry on.